doc/getting-started.md

```diff
@@ -579,10 +633,9 @@ END_TODO
 ### Import base templates
 After you have configured Nexus3, import the base templates for OpenShift.
 Clone the [ocp-templates repository](https://www.github.com/opendevstack/ocp-templates).
```

Comment:
Outdated, this is now `odd-project-quickstarters` repo and then `ocp-templates` folder.